### DISCUSSION TITLE
Function for breaking a block you are looking at or at a point.

### DISCUSSION THREAD

**Discussion by Jibril1453** [PrismarineJS/mineflayer#2603] [Q&A]
Is there any function or method that allows the bot to break the block it is looking at or break the block at a certain point?
I am aware of the bot.dig() function but it takes a block instance as an input, is there anything similar for a point?

**Comment by amoraschi** [comment]
`bot.blockAtCursor()`

Returns the block at the cursor pointer

**Reply by Jibril1453** [reply]
i cant use bot.blockAt() or bot.blockAtCursor() because the server i am playing on does not allow bots to read block data.
